Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Telegrand/Telegrand](https://hosted.weblate.org/projects/telegrand/telegrand/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/telegrand/-/telegrand/horizontal-auto.svg)

just-carlod committed:
Translated using Weblate (Polish)

Currently translated at 100.0% (47 of 47 strings)

Translation: Telegrand/Telegrand
Translate-URL: https://hosted.weblate.org/projects/telegrand/telegrand/pl/

Commit:
Added translation using Weblate (Sinhala)

Commit:
Translated using Weblate (Sinhala)

Currently translated at 70.2% (33 of 47 strings)

Translation: Telegrand/Telegrand
Translate-URL: https://hosted.weblate.org/projects/telegrand/telegrand/si/

marhkb commented on po/pl.po:
```diff
@@ -8,201 +8,205 @@ msgstr ""
 #: src/session/sidebar/chat_row.rs:318
 msgid "You"
-msgstr ""
+msgstr " "
```
Is this whitespace intentional?

melix99 replied:
Well, I don't think the author will ever get notified about this, so I guess the best thing to do here is deleting the string completely so it will appear as untranslated by weblate.

just-carlod replied:
oh sorry I misunderstood the context of this string, you're right it should be "Ty"

danialbehzadi committed:
Translated using Weblate (Persian)

Currently translated at 94.4% (34 of 36 strings)

Translation: Telegrand/Telegrand
Translate-URL: https://hosted.weblate.org/projects/telegrand/telegrand/fa/

just-carlod committed:
Translated using Weblate (Polish)

Currently translated at 100.0% (47 of 47 strings)

Translation: Telegrand/Telegrand
Translate-URL: https://hosted.weblate.org/projects/telegrand/telegrand/pl/

just-carlod commented on po/pl.po:
```diff
@@ -8,201 +8,205 @@ msgstr ""
 #: src/session/sidebar/chat_row.rs:318
 msgid "You"
-msgstr ""
+msgstr " "
```
```suggestion
msgstr "Ty"
```

marhkb commented on po/pl.po:
```diff
@@ -213,11 +217,13 @@ msgstr ""
 msgid "By signing up,\nyou agree to the <a href=\"\">Terms of Service</a>."
 msgstr ""
+"Poprzez założenie konta,\n"
+"akceptujesz <a href=\"\">Zasady Korzystania z Usługi </a>."
```
Should this whitespace be removed?
```suggestion
"akceptujesz <a href=\"\">Zasady Korzystania z Usługi</a>."
```
![Bildschirmfoto von 2021-11-26 21-59-05](https://user-images.githubusercontent.com/3630213/143643727-93709c68-004d-4677-8165-923aef443c54.png)
![Bildschirmfoto von 2021-11-26 21-58-04](https://user-images.githubusercontent.com/3630213/143643714-6cea07e1-05cd-4434-882f-16474a43948b.png)

melix99 replied:
I fixed that from weblate.

abidin251 committed:
Added translation using Weblate (Arabic)

abidin251 committed:
Translated using Weblate (Arabic)

Currently translated at 72.3% (34 of 47 strings)

Translation: Telegrand/Telegrand
Translate-URL: https://hosted.weblate.org/projects/telegrand/telegrand/ar/

abidin251 committed:
Translated using Weblate (Arabic)

Currently translated at 89.3% (42 of 47 strings)

Translation: Telegrand/Telegrand
Translate-URL: https://hosted.weblate.org/projects/telegrand/telegrand/ar/

abidin251 committed:
Translated using Weblate (Arabic)

Currently translated at 91.4% (43 of 47 strings)

Translation: Telegrand/Telegrand
Translate-URL: https://hosted.weblate.org/projects/telegrand/telegrand/ar/

melix99 committed:
Translated using Weblate (Polish)

Currently translated at 100.0% (47 of 47 strings)

Translation: Telegrand/Telegrand
Translate-URL: https://hosted.weblate.org/projects/telegrand/telegrand/pl/
